addons/l10n_account_withholding/tests/test_account_withholding.py

```diff
@@ -0,0 +1,789 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from freezegun import freeze_time
+
+from odoo import Command
+from odoo.addons.account.tests.common import AccountTestInvoicingCommon
+from odoo.addons.analytic.tests.common import AnalyticCommon
+from odoo.exceptions import UserError, RedirectWarning
+from odoo.tests import tagged, Form
+
+
+@tagged('post_install_l10n', 'post_install', '-at_install')
+class TestL10nAccountWithholdingTaxes(AccountTestInvoicingCommon, AnalyticCommon):
+
+    def _setup_tax(self, name, amount, sequence=None, tax_type='sale', price_include='tax_included', base_tag=None, tax_tag=None):
```

Comment:
What about just using the helpers I make in testTaxCommon to create taxes easily?